phost/src/main.rs

```diff
@@ -259,6 +319,59 @@ async fn bridge(args: Args) -> Result<(), Error> {
     }
 }
 
+async fn fetch_events(client: &subxt::Client<Runtime>, block: &phala_node_runtime::SignedBlock, h: u32) -> Result<BlockWithEvents, Error> {
+	let hash = client.block_hash(Some(subxt::BlockNumber::from(NumberOrHex::Number(h)))).await?;
+	let key = storage_value_key_vec("System", "Events");
+	let storage_key = StorageKey(key.clone());
+	let value = get_storage(&client, hash, storage_key.clone()).await.unwrap();
+
+	if filter_events(&client, value.clone()) {
```

Comment:
Why do we need to filter the events? Now it looks like we still pass everything to pruntime